build_and_deploy_liteserv.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+liteserv_branch=$1
+
+if [ "$#" -ne 1 ]; then
+    echo "You must provide 'master' or a branch name for LiteServ"
```

Comment:
@andrei-baranouski I wanted the script to require explicit declaration of the branch to avoid silently checking out master.